tests/nix_tests/configs/default.jsonnet

```diff
@@ -18,6 +18,7 @@
         'logs-cap': 10000,
         'fix-revert-gas-refund-height': 1,
         enable: true,
+        'allow-insecure-unlock': true,
```

Comment:
same here - why is this needed to be set as the default? Is this being used anywhere?
```suggestion
```

Reply:
Because the default value of geth is false, for security reasons, however, the old test files default keyring-test to the open state, so in order for these tests to pass, I have to add 'allow-insecure-unlock': true,